pages/login.html

```diff
@@ -0,0 +1,98 @@
+    <link rel="shortcut icon" href="../assets/logo/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="../stylesheets/style.css">
+</head>
+
+<body>
+    <header class="header-container">
+        <div class="header-brand">
+            <img src="../assets/logo/logo.png" alt="logo">
+            <a href="../index.html" class="header-logo">Compact Store</a>
+        </div>
+        <div class="header-src">
+            <input type="text" name="search" placeholder="What are you looking for?" class="header-src-input">
+            <button type="submit" class="header-src-btn"><i class="fa fa-search"></i></button>
+        </div>
+        <nav class="header-nav">
+            <ul class="header-list">
+                <a href="./signup.html" class="header-item">
+                    <button class="btn btn-primary">Signup</button>
+                </a>
+                <a href="./wishlist.html" class="header-item">
+                    <p class="badge position-relative">
+                        <i class="fa fa-solid fa-heart badge-icon"></i>
+                        <span class="badge-content position-absolute sm-badge">12</span>
+                    </p>
+                    <span class="header-item-text">Wishlist</span>
+                </a>
+                <a href="./cart.html" class="header-item">
+                    <p class="badge position-relative">
+                        <i class="fas fa-shopping-cart badge-icon"></i>
+                        <span class="badge-content position-absolute sm-badge">1</span>
+                    </p>
+                    <span class="header-item-text">Cart</span>
+                </a>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <div class="auth-wrapper rounded-5">
+            <div class="auth-body">
+                <div class="auth-head">
+                    Login
+                </div>
+                <div class="form-container">
+                    <div class="input-container">
+                        <label for="email" class="input-label">Email Address <span
+                                class="required-mark">*</span></label>
+                        <input type="email" id="email" class="input-field" placeholder="xyz@example.com" required>
+                    </div>
+                    <div class="input-container">
+                        <label for="password" class="input-label">Password <span class="required-mark">*</span></label>
+                        <input type="password" for="password" class="input-field" required>
+                    </div>
+                    <div class="auth-form-subsection">
+                        <div class="input-selector">
+                            <input type="checkbox" name="filter" id="" class="input-selector-field" id="checkbox-1">
+                            <label for="checkbox-1" class="input-selector-label">Remember me</label>

```

Comment:
We can put this label before input checkbox

Reply:
+1 to Tahir.
Yes, Vivek I want to have the label after the checkbox.